js/data/locale/en/regionnames.json

```diff
@@ -8127,12 +8179,6 @@
             "name": "Upper Takutu-Upper Essequibo"
         }
     ],
-    "HK": [
```

Comment:
Why is this deleted?

Reply:
To generated the data, The tool uses the xml file (common/subdivisions/*.xml).  HK part is deleted from the en.xml file. 
`<subdivision type="HK">Hong Kong SAR China</subdivision>`